csrf.go

```diff
@@ -17,10 +17,11 @@ const tokenLength = 32
 
 // Context/session keys & prefixes
 const (
-	tokenKey    string = "goji.csrf.Token"
-	errorKey    string = "goji.csrf.Error"
-	cookieName  string = "_goji_csrf"
-	errorPrefix string = "goji/csrf: "
+	tokenKey     string = "goji.csrf.Token"
+	fieldNameKey string = "goji.csrf.FieldName"
```

Comment:
Sorry to do this (naming things is the worst) - but let's change this to `formKey` (which is what it is) and make it `goji.csrf.Token` by default. This describes what the `<input>` field contains.